sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -807,6 +819,17 @@ public partial class ImageAnalysisSkill : Azure.Search.Documents.Indexes.Models.
         public static bool operator !=(Azure.Search.Documents.Indexes.Models.ImageDetail left, Azure.Search.Documents.Indexes.Models.ImageDetail right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class IndexerCurrentState
```

Comment:
This should probably just be called `IndexerState` and then the property name can indicate the "current" nature of the result.

Reply:
There is no property talking about the 'current' state. The 'initial' and 'final' states are updated as appropriate.

Reply:
`IndexerExecutionResult` has a property of this type.  I assume that's meant to be "current"?